documentation/modules/ROOT/pages/connectors/db2.adoc

```diff
@@ -0,0 +1,1503 @@
+| Represents the number of milliseconds since epoch, and does not include timezone information. Db2 allows `P` to be in the range 0-7 to store up to tenth of microsecond precision, though this mode results in a loss of precision when `P` > 3.
+
+|=======================
+
+[[timestamp-values]]
+===== Timestamp values
+
+The `DATETIME`, `SMALLDATETIME` and `DATETIME2` types represent a timestamp without time zone information.
+Such columns are converted into an equivalent Kafka Connect value based on UTC.
+So for instance the `DATETIME2` value "2018-06-20 15:13:16.945104" will be represented by a `io.debezium.time.MicroTimestamp` with the value "1529507596945104".
+
+Note that the timezone of the JVM running Kafka Connect and Debezium does not affect this conversion.
+
+==== Decimal values
+
+[cols="15%a,15%a,35%a,35%a",width=100,options="header,footer",role="table table-bordered table-striped code-wordbreak-col3  code-wordbreak-col4"]
+|=======================
+|Db2 Data Type
+|Literal type (schema type)
+|Semantic type (schema name)
+|Notes
+
+|`NUMERIC[(P[,S])]`
+|`BYTES`
+|`org.apache.kafka.connect.data.Decimal`
+|The `scale` schema parameter contains an integer representing how many digits the decimal point was shifted.
+The `connect.decimal.precision` schema parameter contains an integer representing the precision of the given decimal value.
+
+|`DECIMAL[(P[,S])]`
+|`BYTES`
+|`org.apache.kafka.connect.data.Decimal`
+|The `scale` schema parameter contains an integer representing how many digits the decimal point was shifted.
+The `connect.decimal.precision` schema parameter contains an integer representing the precision of the given decimal value.
+
+|`SMALLMONEY`
+|`BYTES`
+|`org.apache.kafka.connect.data.Decimal`
+|The `scale` schema parameter contains an integer representing how many digits the decimal point was shifted.
+The `connect.decimal.precision` schema parameter contains an integer representing the precision of the given decimal value.
+
+|`MONEY`
+|`BYTES`
+|`org.apache.kafka.connect.data.Decimal`
+|The `scale` schema parameter contains an integer representing how many digits the decimal point was shifted.
+The `connect.decimal.precision` schema parameter contains an integer representing the precision of the given decimal value.
+
+|=======================
+
+[[deploying-a-connector]]
+== Deploying a connector
+
+If you've already installed https://zookeeper.apache.org[Zookeeper], http://kafka.apache.org/[Kafka], and http://kafka.apache.org/documentation.html#connect[Kafka Connect], then using Debezium's Db2` connector is easy.
+Simply download the https://repo1.maven.org/maven2/io/debezium/debezium-connector-sqlserver/0.9.0.Alpha1/debezium-connector-sqlserver-0.9.0.Alpha1-plugin.tar.gz[connector's plugin archive], extract the JARs into your Kafka Connect environment, and add the directory with the JARs to http://docs.confluent.io/{confluent-platform-version}/connect/userguide.html#installing-plugins[Kafka Connect's classpath].
```

Comment:
So this also answers one of my questions above; here the procedure for downloading and adding the JDBC driver should be described.

Reply:
See previous comments.